### DISCUSSION TITLE
[Feature Request] Support OpenSearch Cross-Cluster Search Capability

### DISCUSSION THREAD

**Discussion by rafael-gumiero** [grafana/grafana#53425] [Data Sources]
I currently use [OpenSearch's cross-cluster search](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/cross-cluster-search.html) capability to be able to perform queries on multiple clusters through a single point. These queries are done through "connection alias". I would like to be able to use this same capability through the OpenSearch datasource via Grafana.

Example:
```
GET https://src-domain.us-east-1.es.amazonaws.com/<connection_alias>:books/_search

{
    ...
  "hits": [
    {
      "_index": "source-destination:books",
      "_type": "_doc",
      "_id": "1",
      "_score": 1,
      "_source": {
        "Dracula": "Bram Stoker"
      }
    }
  ]
}
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
